EcoSoapBank/Helpers/UserDefaults.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  UserDefaults.swift
+//  EcoSoapBank
+//
+//  Created by Shawn Gee on 8/18/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import Foundation
+
+/// This is where all of our user defaults can be stored. This setup uses a
+/// property wrapper which makes it even simpler to get and set user defaults.
+/// You can also observe changes on user defaults using a UserDefaultsObservation.
+extension UserDefaults {
+    @UserDefault(Key("massUnit")) static var massUnit: String?
```

Comment:
Might be worth making some constant `static` properties on `Key` for the keys we'll use for this? You might have already thought of that but thought I'd mention it if not. Could even be useful in other parts of the app, since the name `Key` implies a high level of genericness.

Reply:
I did do that initially (that's what the author did in the article), but then just did it like this because it seemed redundant since this itself is a static property (which actually should be a constant, I'll go change that).

Reply:
Ah, that's true I suppose.